internal/runtime/graphviz/graphviz.go

```diff
@@ -0,0 +1,127 @@
+package graphviz
```

Comment:
please move this to `compiler/backend/dot` package, runtime should not know anything about graphviz